src/API/Management/ResourceServers.php

```diff
@@ -53,6 +71,11 @@ public function update($id, $data)
     {
         $response = $this->httpClient->patch(sprintf('/resource-servers/%s', $id), [], json_encode($data));
 
-        return ResponseMediator::getContent($response);
+        // TODO Is this really correct?
+        if (201 === $response->getStatusCode()) {
+            return ResponseMediator::getContent($response);
```

Comment:
Well if docs say it I would hope so, although agree it's inconsistent with other `update` endpoints.  @glena ?